saos-webapp/src/main/resources/message/common.properties

```diff
@@ -288,6 +294,13 @@ courtCriteriaFormSection.hint.administrativeCourt.content = Dane nie zostały na
 
 
 /* Judgment details view */
+details.meta.judgment = Orzeczenie
+details.meta.decisionReleasedBy = wydane przez
```

Comment:
I suggest to name these: releasedBy.neuter, releasedBy.feminine, releasedBy.masculine